lib/app_web/templates/tag/index.html.heex

```diff
@@ -13,7 +13,7 @@
         <.td>
           <span
             style={"background-color:#{tag.color}"}
-            class="w-40 text-white font-bold py-1 px-2 rounded-full 
+            class="max-w-[144px] text-white font-bold py-1 px-2 rounded-full 
```

Comment:
Hmm… why 144px? Is this tested on any mobile devices? 📱

Reply:
yes tested on mobile.
I've take the value to match the `m-36` (m-40 was a bit too long I think)
![image](https://user-images.githubusercontent.com/6057298/204568381-54cedbf4-6e27-4be9-90df-1e02d4208302.png)

https://tailwindcss.com/docs/width

Reply:
OK. so you had to hard-code it because `max-w-xs` is **`320px`** 💭 
![image](https://user-images.githubusercontent.com/194400/204598781-94fde3f8-0890-4d12-b5ab-6092b1fb8b95.png)

Reply:
So yes, I started tested with `max-w-xs`, it was looking ok on desktop:
![image](https://user-images.githubusercontent.com/6057298/204750406-76a656c2-0d56-4226-870c-f77888efc2e3.png)

However when I tested on mobile, the table wasn't displayed in full:

![image](https://user-images.githubusercontent.com/6057298/204750525-dbbe221c-a98c-43ca-8fc0-85acefeb8b55.png)

I knew with my previous PR(https://github.com/dwyl/mvp/pull/218) that using the fixed width `w-40` the display of the table was working on both desktop and mobile.

However Tailwind doesn't provide the `max-w-40`. I found the [arbitrary value](https://tailwindcss.com/docs/max-width#arbitrary-values) section in the documentation and decided to use then the `max-w-[160px]` first to match the `w-40`. However I still find the table/tag pills a bit too long and went with `max-w-[144px]` to match the `w-36`

Test describe in the comment https://github.com/dwyl/mvp/issues/217#issuecomment-1330728897 while I was doing the work on the PR

Reply:
Cool. Thanks for sharing more detail. 👌